Islandora-Devops/claw-playbook
# Paged content config

**GitHub Issue**: https://github.com/Islandora-CLAW/CLAW/issues/932

# What does this Pull Request do?

This pulls together all the config needed for the work from the Paged Content sprint.  It ties together all the great work from everyone.  Plus it installs a few extra things we've collected over time but aren't installed by default (oai-pmh and breadcrumbs).  Also: `admin_toolbar`.

# What's new?
Paged content!

# How should this be tested?

- Pull down this PR
- `vagrant up`
- Make a 'Repository Item' and give it a model of 'Paged Content'
- Add some children to it
  - Give each one a model of 'Page'
  - Give each one a TIFF or JPEG for an 'Original File'
- Clear your cache
- Visit the 'Paged Content' item and you should see openseadragon pop up with arrows to go left/right through the pages.  You should also see a 'reference strip' at the bottom of the viewer so you can see all the pages and where you are in the sequence.
- From your 'Paged Content' node, click its Children tab, and then the "Reorder Children" button
- Re-order the pages using the drag and drop
- Clear your cache
- Visit your 'Paged Content' node again, and the order of the pages should be reflected.

# Interested parties
Fun one to test that shows all the great work the community has done!  @Islandora-Devops/committers @rangel35 @dbernstein @elizoller @manez
